django/apps/registry.py

```diff
@@ -156,6 +156,9 @@ def get_app_config(self, app_label):
                 if app_config.name == app_label:
                     message += " Did you mean '%s'?" % app_config.label
                     break
+            else:
```

Comment:
I don't think this addition is needed. It the case of the original report, the existing "Did you mean" is enough. This new "Did you mean" won't actually work because the app isn't in INSTALLED_APPS.

Reply:
The original test is faulty. It tests a nonexistent dotted path rather than one that exists and is in INSTALLED_APPS.

Reply:
`django.contrib.auth` is not in INSTALLED_APPS for those tests because of `available_apps = ["migrations"]`, I believe. What I'm trying to say is this new message is misleading because acting on the suggestion isn't sufficient to fix the problem.